### PR TITLE
global: addition of ``submissions`` module 

### PR DESCRIPTION
**Pull request by drjova**

### CODE REVIEW

**Comment by drjova** [comment]
ci test please

**Comment by drjova** [comment]
ci test please

File: inspirehep/modules/submissions/serializers/schemas/author.py [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+            ids = advisor.get('ids')
+            record = advisor.get('record')
+            author.add_advisor(
+                name, ids=ids, degree_type=degree_type, record=record)
+
+        for arxiv_category in data.get('arxiv_categories', []):
+            author.add_arxiv_category(arxiv_category)
+
+        blog = data.get('blog')
+        if blog:
```
[the remark this answers is not in the document]
Comment (harunurhan) [reply inspirehep/modules/submissions/serializers/schemas/author.py]: that's nice! I think by the time I did this part, it wasn't

File: inspirehep/modules/submissions/serializers/schemas/author.py [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+
+        acquisition_source = data.get('acquisition_source')
+        if acquisition_source:
+            author.add_acquisition_source(**acquisition_source)
+
+        return author.obj
+
+    def get_full_name(self, given_name, family_name):
+        if given_name and family_name:
+            return '{} {}'.format(given_name, family_name)
```
[the remark this answers is not in the document]
Comment (harunurhan) [reply inspirehep/modules/submissions/serializers/schemas/author.py]: according to serializer login, shouldn't be `<given_name> <family_name>`

File: inspirehep/modules/submissions/serializers/schemas/author.py [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+            'websites': get_value(data, 'urls.value', default=missing),
+        })
+
+        return author_dict
+
+    def get_name_splitted(self, data):
+        name = get_value(data, 'name.value')
+        if name:
+            name = name.replace(',', '')
+            first, last = name.split(' ')
```
[the remark this answers is not in the document]
Comment (michamos) [reply inspirehep/modules/submissions/serializers/schemas/author.py]: It depends on which field is mandatory. It should go to the mandatory one.

**Comment by drjova** [comment]
@michamos updated!

File: inspirehep/modules/submissions/serializers/schemas/author.py [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+    comments = fields.Raw()
+
+    acquisition_source = fields.Raw()
+
+    @pre_dump
+    def before_dump(self, data):
+        author_dict = {}
+
+        family_name, given_name = self.get_name_splitted(data)
+        author_dict.update({
```
Comment (michamos) [inline inspirehep/modules/submissions/serializers/schemas/author.py]: what's the difference between doing this and returning directly the argument to `update`? here it looks like you are building two `dict`s for no good reason.

**Comment by drjova** [comment]
ci test please

**Comment by drjova** [comment]
ci test please

**Comment by drjova** [comment]
@michamos I have to close this one due to the ``ui_authors`` setting on jenkins. I reopen it as https://github.com/inspirehep/inspire-next/pull/3660

**Pull request closed without merging**